openedx_proversity_reports/reports/last_page_accessed.py

```diff
@@ -34,29 +36,35 @@ def get_last_page_accessed_data(course_list):
         except InvalidKeyError:
             continue
 
-        # Getting all students enrolled in the course except staff users
-        enrolled_students = User.objects.filter(
+        # Getting all users enrolled in the course.
+        enrolled_users = User.objects.filter(
             courseenrollment__course_id=course_key,
             courseenrollment__is_active=1,
-            is_staff=0,
         )
 
-        if not enrolled_students:
+        if not enrolled_users:
             continue
 
         user_data = []
-
         usage_key = get_modulestore().make_course_usage_key(course_key)
-        blocks = get_course_blocks(enrolled_students[0], usage_key)
+        blocks = get_course_blocks(enrolled_users[0], usage_key)
```

Comment:
if the user is instructor or student, does it return the same ?, else you should ensure that the user has access to the whole course

Reply:
In fact, the first user enrolled in the curse always is the staff user.

Reply:
ok